From: Jan Engelhardt <ej@inai.de>
Date: 2024-11-02 16:06:53.256545988 +0100
References: https://github.com/gphalkes/t3key/pull/2

.config.c:8:13: error: implicit declaration of function ‘setupterm’; did you mean ‘set_term’? [-Wimplicit-function-declaration]

---
 config.pkg |    1 +
 1 file changed, 1 insertion(+)

Index: libt3key-0.2.10/config.pkg
===================================================================
--- libt3key-0.2.10.orig/config.pkg
+++ libt3key-0.2.10/config.pkg
@@ -144,6 +144,7 @@ EOF
 #include <stdio.h>
 #include <stdlib.h>
 #include <curses.h>
+#include <term.h>
 
 int main(int argc, char *argv[]) {
 	int args[9], error, fd;
